tools/fp20compiler/ts1.0_inst_list.cpp

```diff
@@ -219,6 +219,61 @@ void InstList::Invoke()
     printf(");\n");
     printf("p += 2;\n");
 
+    // Process texture stage mode arguments

```

Comment:
Consider adding a case for `TSP_CULL_FRAGMENT` with `assert(false); /* Unimplemented */` (and then add the `/* Untested */` to the `assert(false)` in the switch above for consistency)